nft-block.js

```diff
@@ -88,10 +81,46 @@ module.exports = class NftBlock extends Block {
     
     // Adding NFT to artists list.
     let ownedNfts = this.nftOwnerMap.get(owner) || [];
-    ownedNfts.push(nftID);
-    this.nftOwnerMap.set(owner, ownedNfts);
+    if(ownedNfts.includes(nftID) === false)
+    {
+      ownedNfts.push(nftID);
+      this.nftOwnerMap.set(owner, ownedNfts);
+    }
   }
-///////////////////////////
+
+  transferNft(sender, receiver, title, artName) {
```

Comment:
Looks like you have competing transferNft methods.  Delete out whichever one is not being used.